### ISSUE TITLE
Bad UX: Wrong password does not show login error prior to 2FA screen

### ISSUE THREAD

**Issue by roschaefer** [tactilenews/100eyes#1162]
When I logged in recently and my password was changed, I would see an error message only after I entered the 2FA token. This would make me believe that the 2FA token is incorrect, not the password.

**Comment by tillprochaska** [comment]
Ich kann das leider nicht nachvollziehen, kannst du Schritte zum Reproduzieren ergänzen?

https://user-images.githubusercontent.com/1512805/142827009-56e7f3a9-8ab4-426b-a49a-5ea9d2bacca2.mov

**Comment by roschaefer** [comment]
On https://staging.100ey.es/sign_in I see this after I enter my username/password combination and the 2FA token:
![image](https://user-images.githubusercontent.com/2110676/144723347-a7417453-5f4a-4539-a00d-6ca279d2e805.png)

I checked in my password manager and in via `bin/rails c` on the VPS server and the OTP secret is the same.

I just reset my password and the problem persists. I copied the `otp_secret_key` into my password manager again and it still doesn't work.
